services/cd-service/pkg/service/batch.go

```diff
@@ -199,11 +201,10 @@ func (d *BatchServer) processAction(
 			return nil, nil, status.Error(codes.InvalidArgument, "invalid Team name")
 		}
 		return &repository.ReleaseTrain{
-				Target:         in.Target,
-				Team:           in.Team,
-				Authentication: repository.Authentication{RBACConfig: d.RBACConfig},
-				Repo:           d.Repository,
-				CommitHash:     in.CommitHash,
+				Target:          in.Target,
+				Team:            in.Team,
+				WriteCommitData: d.Config.WriteCommitData,
+				Authentication:  repository.Authentication{RBACConfig: d.RBACConfig},
```

Comment:
why are we removing the `CommitHash` parameter here? Sounds like a bug.

Reply:
Yup, that looks like a bug introduced during the merge.

However, I am confused as to why a commit hash is being passed as part of a release train, but this is a conversation for later.

Reply:
What's also interesting is that unit tests passed with this bug introduced, so maybe we have some more testcases to write.